ibex_configs.yaml

```diff
@@ -37,6 +38,7 @@ opentitan:
   PMPGranularity           : 0
   PMPNumRegions            : 16
   SecureIbex               : 1
+  Scramble                 : 0
```

Comment:
Sensible, but could you file an issue (on the Ibex repo) to turn this on once we think it works?